app/views.py

```diff
@@ -289,12 +289,25 @@ def finalize_or_reject(request, process_id):
     user = request.data["authorized"]
     user_type = request.data["user_type"]
     state = request.data["action"]
+
+    if state == "rejected":
+        message = request.data.get("rejectionMsg", None)
```

Comment:
change this request body key to the same naming style as other variables "rejection_message"

Reply:
Done.